boxoffice/templates/boxoffice.js

```diff
@@ -242,7 +242,7 @@ $(function() {
               var line_items = boxoffice.ractive.get('order.line_items');
               line_items.forEach(function(line_item) {
                 if (data.line_items.hasOwnProperty(line_item.item_id)) {
-                  if(data.line_items[line_item.item_id].discounted_amount && line_item.quantity_available > 0) {
+                  if (data.line_items[line_item.item_id].discounted_amount && line_item.quantity_available > 0) {
                     valid_discount_coupon = true;
```

Comment:
Depending on the value of `valid_discount_coupon`, `order.line_items` is set and page is scrolled to the Boxoffice widget (Line: 257)